aries-site/src/pages/components/grid.mdx

```diff
@@ -13,3 +22,201 @@ import { GridExample } from '../../examples';
+a variety of browser widths.
+- Fluid grids allowing columns and rows to grow and adapt to the content provided.
+- Precise placement of elements within the grid via [areas](#areas).
+
+## Grid Properties & Key Concepts
+Grid's primary properties include: [columns](#columns), [rows](#rows), and [areas](#areas). The 
+following examples will demonstrate how these properties may be applied to affect 
+[content layouts](/templates/content-layouts).
+
+### Columns
+Columns may be specified as [uniform widths](#uniform-columns), [mixed widths](#mixed-width-columns), 
+or as [min-max ranges](#flexible-columns) which allow the contents to flex gracefully.
+
+#### Uniform Columns
+Uniform columns are ideal for presenting a collection of like items such as a grid of images, 
+a catalog of services, or a [dashboard of cards](/templates/dashboards#three-column-dashboard).
+
+Specifying the column width as single string will repeat multiple columns of that size, as long as 
+there is room for more. For example, `columns="medium"` will fit as many "medium" columns into the 
+grid as the overall width allows. Within a wide screen, this might allow 3 or 4 columns; within a narrow 
+screen, maybe only 1 or 2.
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/UniformColumns.js"
+  width={{ min: 'medium' }}
+>
+  <UniformColumns />
+</Example>
+
+#### Mixed-Width Columns
+Mixed-width columns allow for control over layouts desiring columns of various widths to appropriately present 
+their contents, establish content hierarchy with wider columns for primary content and narrower for secondary 
+content such as an [activity feed](/templates/dashboards#two-column-dashboard).
+
+Mixed-width columns are specified in an array. The number of items in the array dictate the number of columns 
+in the grid. 
+
+For example, columns specified as `columns={[ 'small', 'xsmall', 'xsmall' ]}` would create a three 
+column grid with corresponding widths: 
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/MixedColumns1.js"
+  width={{ min: 'medium' }}
+  height="auto"
+>
+  <MixedColumns1 />
+</Example>
+
+<br /><br />
+
+And, `columns={[ '2/3', '1/3' ]}` would result in two columns with proportional 
+widths:
+
+<Example
+  horizontalLayout
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/feat-2381-grid-docs/aries-site/src/examples/components/grid/MixedColumns2.js"
+  width={{ min: 'medium' }}
+  height="auto"
+>
+  <MixedColumns2 />
+</Example>
+
+
+#### Flexible Columns
+Flexible columns is where Grid really shines. They allow content to elegantly scale and designs to be dynamic.
+
+The main idea is defining columns as a min-max range. Between those bounds, the column is allowed to 
```

Comment:
The column is allowed to flex as large as the max and as small as the min, depending on its content and the space available to the grid as a whole.

Reply:
Great suggestion. Done.